en/04_Changelogs/5.0.0.md

```diff
@@ -732,13 +732,13 @@ This is a major release and contains many breaking API changes. Deprecation warn
 
 #### silverstripe/cms
 
+- Class `SilverStripe\CMS\Controllers\SilverStripeNavigator` has been moved from `silverstripe/cms` to `silverstripe/admin` and renamed as `SilverStripe\Admin\Navigator\SilverStripeNavigator`
+- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem` has been moved from `silverstripe/cms` to `silverstripe/admin` and renamed as `SilverStripe\Admin\Navigator\SilverStripeNavigatorItem`
+- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_Unversioned` has been moved from `silverstripe/cms` to `silverstripe/admin` and renamed as `SilverStripe\Admin\Navigator\SilverStripeNavigatorItem_Unversioned`
+- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_ArchiveLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink`
+- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_LiveLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_LiveLink`
+- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_StageLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed as `SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_StageLink`
```

Comment:
```suggestion
- Class `SilverStripe\CMS\Controllers\SilverStripeNavigator` has been moved from `silverstripe/cms` to `silverstripe/admin` and renamed as [`SilverStripe\Admin\Navigator\SilverStripeNavigator`](api:SilverStripe\Admin\Navigator\SilverStripeNavigator)
- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem` has been moved from `silverstripe/cms` to `silverstripe/admin` and renamed as [`SilverStripe\Admin\Navigator\SilverStripeNavigatorItem`](api:SilverStripe\Admin\Navigator\SilverStripeNavigatorItem)
- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_Unversioned` has been moved from `silverstripe/cms` to `silverstripe/admin` and renamed as [`SilverStripe\Admin\Navigator\SilverStripeNavigatorItem_Unversioned`](api:SilverStripe\Admin\Navigator\SilverStripeNavigatorItem_Unversioned)
- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_ArchiveLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed as [`SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink`](api:SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink)
- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_LiveLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed as [`SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_LiveLink`](api:SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_LiveLink)
- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_StageLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed as [`SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_StageLink`](api:SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_StageLink)
```
We use links to API docs for any classes/methods that are still around.

Reply:
Updated 👍